style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-size: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+@import url(https://fonts.googleapis.com/css?family=roboto);
+
+html {
+  font-family: "roboto", arial;
+  font-size: 18px;
+}
+
+nav {
+  justify-content: center;
+  padding: 20px;
+  background-color: #046c94;
+  color: #fff;
+  list-style-type: none;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-links li {
+  list-style-type: none;
+}
+
+a {
+  color: #fff;
+  text-decoration: underline;
+}
+
+body, html {
+  height: 100%;
+  margin: 0;
+  background-color: #000;
+  color: #fff;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url("https://images.unsplash.com/photo-1555099962-4199c345e5dd?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1350&q=80");
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  font-size: 25px;
+}
+
+.hero-text button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 10px 25px;
+  color: #fff;
+  background-color: #046c94;
+  text-align: center;
+  cursor: pointer;
+}
+
+.hero-text button:hover {
+  background-color: #34c2b2;
+  color: #fff;
+}
+
+section {
+  width: 800px;
```

Comment:
This width works really nicely on desktop, but on small mobile devices it causes the content to be wider than the device.

There is actually a really nice fix for this issue which is to give the section a max width as well, like this:

```
section {
    width: 800px;
    max-width: 95%;
}
```

That way, the section will be 800px, unless that is greater than 95% of the total screen width, in which case it will always take up the 95%.

Reply:
Great to know, many thanks =]